common/app/views/fragments/commonCssSetup.scala.html

```diff
@@ -86,5 +86,50 @@
   - Note: does not include IE10 on Windows Phone 8 (which does not support conditional comments)
 *@
 <!--[if IEMobile]>
-    <link rel="stylesheet" type="text/css" href="@Static("stylesheets/global.css")" />
+    <link rel="stylesheet" type="text/css" href="@Static("stylesheets/global.css")" media="only x" />
 <![endif]-->
+
+@*
+  - Include any browser [<!-->]
+  - Exclude IE < 10 [(gt IE 9)]
+  - Exclude IE Mobile [&(!IEMobile)]
+*@
+<!--[if (gt IE 9)&(!IEMobile)]><!-->
+    <link rel="stylesheet" data-reload="global" type="text/css" href="@Static("stylesheets/global.css")" media="only x" />
+    <noscript>
+        <link rel="stylesheet" type="text/css" href="@Static("stylesheets/global.css")" />
+    </noscript>
+<!--<![endif]-->
+
+<script>
+    // The css is loaded by the `link`s above but with media of "only x".
+    // This means the browser requests it, but doesn't block rendering on it,
+    // since this media is not 'x'. Once it's loaded (appears in
+    // `window.document.styleSheets`),  `media` is set to `all` so the browser
+    // uses it. Very much inspired by https://github.com/filamentgroup/loadCSS.
+    (function (isModernEnough) {
```

Comment:
as discussed with @piuccio, `querySelectorAll` is ok in IE8 with CSS2 selectors, but `bind` is a def no-no - stupid. Now using a `function` in the `setTimeout` to pass the arguments back